src/orion/core/io/resolve_config.py

```diff
@@ -37,6 +37,7 @@
 import socket
 
 from numpy import inf as infinity
+
```

Comment:
There should be no space here, flake8 complains.

Reply:
I don't get it, your linter doesn't like any order. With spaces or without.